uaclient/entitlements/base.py

```diff
@@ -245,6 +251,26 @@ def process_contract_deltas(
             # responses on unentitled services.
             self.cfg.delete_cache_key('machine-access-%s' % self.name)
             return True
+
+        resourceToken = orig_access.get('resourceToken')
```

Comment:
Shuffled this up into base and out of RepoEntitlement so that livepatch can be auto-enabled by process_contract_deltas.